### ISSUE TITLE
[NPC] Tunnel Rat Kobolds near Thelsamar have unusual attack speed (way too fast)

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5357]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
While killing Tunnel Rat Kobolds for the quest "Rat Catching", I've noticed that shortly after the initial engagement these mobs will start to attack at crazy speeds of 1 attack per second or faster (without any form of attack speed buff which they should not have anyway).X

**Expected behaviour**: Tell us what should happen instead
The kobolds should be attacking at the standard speed of about 1 attack per 2 seconds.

**Steps to reproduce the problem**:

1. Go directly west of Thelsamar till you find a slope down leading to a kobold camp
2. Engage the Tunnel Rat Kobolds in combat, allowing them to land melee hits on you
3. Observe the damage taken which is indicative of the melee attacks they deal. You can also observe the attack speed trend from the combat log.

**Include proofs for this behaviour**
![image](https://user-images.githubusercontent.com/32976781/37767034-b6766246-2e03-11e8-87c1-f047d1dfcd4b.png)
Not sure if I managed to upload correctly but this is supposedly a screenshot of the combat log as proof. I couldn't take one fast enough in real time during combat but the log shows a whole string of kobold attacks in between my attacks.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
yes the reason for the issue is the spell 'trash' https://wowgaming.altervista.org/aowow/?spell=12787, which is an aura. This aura is several times on the creature and stacks. The aura should be there only one time.

**Comment by Rushor** [comment]
related https://github.com/dalaranwow/dalaran-wow/issues/2180

**Comment by Rushor** [comment]
The spell 'Thrash' should no longer stack for all creature which use this.

Creature ID List

https://wowgaming.altervista.org/aowow/?npc=23680 
https://wowgaming.altervista.org/aowow/?npc=11372 
https://wowgaming.altervista.org/aowow/?npc=14821 
https://wowgaming.altervista.org/aowow/?npc=14882 
https://wowgaming.altervista.org/aowow/?npc=11357 
https://wowgaming.altervista.org/aowow/?npc=14750 
https://wowgaming.altervista.org/aowow/?npc=9096  
https://wowgaming.altervista.org/aowow/?npc=449   
https://wowgaming.altervista.org/aowow/?npc=2560  
https://wowgaming.altervista.org/aowow/?npc=1353  
https://wowgaming.altervista.org/aowow/?npc=4479  
https://wowgaming.altervista.org/aowow/?npc=2728  
https://wowgaming.altervista.org/aowow/?npc=5977  
https://wowgaming.altervista.org/aowow/?npc=478   
https://wowgaming.altervista.org/aowow/?npc=6005  
https://wowgaming.altervista.org/aowow/?npc=8299  
https://wowgaming.altervista.org/aowow/?npc=7039  
https://wowgaming.altervista.org/aowow/?npc=8981  
https://wowgaming.altervista.org/aowow/?npc=9176  
https://wowgaming.altervista.org/aowow/?npc=9777  
https://wowgaming.altervista.org/aowow/?npc=7369  
https://wowgaming.altervista.org/aowow/?npc=2717  
https://wowgaming.altervista.org/aowow/?npc=5435  
https://wowgaming.altervista.org/aowow/?npc=4023  
https://wowgaming.altervista.org/aowow/?npc=5455  
https://wowgaming.altervista.org/aowow/?npc=5456  
https://wowgaming.altervista.org/aowow/?npc=5457  
https://wowgaming.altervista.org/aowow/?npc=5458  
https://wowgaming.altervista.org/aowow/?npc=5459  
https://wowgaming.altervista.org/aowow/?npc=5460  
https://wowgaming.altervista.org/aowow/?npc=7234  
https://wowgaming.altervista.org/aowow/?npc=3256  
https://wowgaming.altervista.org/aowow/?npc=3378  
https://wowgaming.altervista.org/aowow/?npc=100   
https://wowgaming.altervista.org/aowow/?npc=1202  
https://wowgaming.altervista.org/aowow/?npc=1783  
https://wowgaming.altervista.org/aowow/?npc=1791  
https://wowgaming.altervista.org/aowow/?npc=4493  
https://wowgaming.altervista.org/aowow/?npc=519   
https://wowgaming.altervista.org/aowow/?npc=3722  
https://wowgaming.altervista.org/aowow/?npc=3746  
https://wowgaming.altervista.org/aowow/?npc=10639 
https://wowgaming.altervista.org/aowow/?npc=12940 
https://wowgaming.altervista.org/aowow/?npc=14282 
https://wowgaming.altervista.org/aowow/?npc=14283 
https://wowgaming.altervista.org/aowow/?npc=13088 
https://wowgaming.altervista.org/aowow/?npc=10991 
https://wowgaming.altervista.org/aowow/?npc=16216 
https://wowgaming.altervista.org/aowow/?npc=8138  
https://wowgaming.altervista.org/aowow/?npc=7268  
https://wowgaming.altervista.org/aowow/?npc=3636  
https://wowgaming.altervista.org/aowow/?npc=11043 
https://wowgaming.altervista.org/aowow/?npc=10414 
https://wowgaming.altervista.org/aowow/?npc=10416 
https://wowgaming.altervista.org/aowow/?npc=10407 
https://wowgaming.altervista.org/aowow/?npc=4540  
https://wowgaming.altervista.org/aowow/?npc=12258 
https://wowgaming.altervista.org/aowow/?npc=11782 
https://wowgaming.altervista.org/aowow/?npc=16593 
https://wowgaming.altervista.org/aowow/?npc=17397 
https://wowgaming.altervista.org/aowow/?npc=29392 
https://wowgaming.altervista.org/aowow/?npc=29033 
https://wowgaming.altervista.org/aowow/?npc=18631 
https://wowgaming.altervista.org/aowow/?npc=1696  
https://wowgaming.altervista.org/aowow/?npc=4855  
https://wowgaming.altervista.org/aowow/?npc=5267  
https://wowgaming.altervista.org/aowow/?npc=7320  
https://wowgaming.altervista.org/aowow/?npc=7446  
https://wowgaming.altervista.org/aowow/?npc=9164  
https://wowgaming.altervista.org/aowow/?npc=9167  
https://wowgaming.altervista.org/aowow/?npc=10488 
https://wowgaming.altervista.org/aowow/?npc=11442 
https://wowgaming.altervista.org/aowow/?npc=11486 
https://wowgaming.altervista.org/aowow/?npc=11677 
https://wowgaming.altervista.org/aowow/?npc=12138 
https://wowgaming.altervista.org/aowow/?npc=13285 
https://wowgaming.altervista.org/aowow/?npc=639   
https://wowgaming.altervista.org/aowow/?npc=18105 
https://wowgaming.altervista.org/aowow/?npc=5709  
https://wowgaming.altervista.org/aowow/?npc=3456  
https://wowgaming.altervista.org/aowow/?npc=9267

**Comment by wrathofthebug** [comment]
Thanks for the update and insight. So I presume a fix for this to prevent stacking of the "Thrash" aura is in the works? Because I've actually noticed this stacking bug on other mobs as well in the past now that you mentioned it.

**Comment by Rushor** [comment]
yes it's waiting for ptr update

**Comment by wrathofthebug** [comment]
Hi it's been more than a month may I know if there are any updates to the progress of this fix? One of my alts just fought a defias knuckleduster and it seems like the bug has yet to be fixed.
